packages/ra-input-rich-text/src/RichTextInput.tsx

```diff
@@ -1,26 +1,26 @@
-import * as React from 'react';
-import { ReactElement, ReactNode, useEffect } from 'react';
-import clsx from 'clsx';
-import { useEditor, Editor, EditorOptions, EditorContent } from '@tiptap/react';
-import StarterKit from '@tiptap/starter-kit';
-import Underline from '@tiptap/extension-underline';
+import { FormHelperText } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { Color } from '@tiptap/extension-color';
+import Highlight from '@tiptap/extension-highlight';
+import Image from '@tiptap/extension-image';
 import Link from '@tiptap/extension-link';
 import TextAlign from '@tiptap/extension-text-align';
-import Image from '@tiptap/extension-image';
 import TextStyle from '@tiptap/extension-text-style';
-import { Color } from '@tiptap/extension-color';
-import Highlight from '@tiptap/extension-highlight';
-import { FormHelperText } from '@mui/material';
-import { styled } from '@mui/material/styles';
+import Underline from '@tiptap/extension-underline';
+import { Editor, EditorContent, EditorOptions, useEditor } from '@tiptap/react';
+import StarterKit from '@tiptap/starter-kit';
+import clsx from 'clsx';
 import { useInput, useResourceContext } from 'ra-core';
 import {
     CommonInputProps,
     InputHelperText,
     Labeled,
     LabeledProps,
 } from 'ra-ui-materialui';
-import { TiptapEditorProvider } from './TiptapEditorProvider';
+import * as React from 'react';
```

Comment:
I don't know why you did that, but every React file in react-admin has this line as first and I don't think we should change it.